src/index.ts

```diff
@@ -0,0 +1,70 @@
+import express from "express";
+import bodyParser from "body-parser";
+import { createClient } from "redis";
+import "./config";
```

Comment:
this is a file? or? configs should probably be stored in a JSON file if not sensitive or .env for sensitive

Reply:
No it's just stuff that runs on startup